pkg/koordlet/executor/resource_update_executor.go

```diff
@@ -219,6 +219,8 @@ func (e *LeveledResourceUpdateExecutor) LeveledUpdateBatchByCache(resources [][]
 			}
 		}
 	}
+
+	updated = err == nil
```

Comment:
i suggest we can remove the return value since the meaning is alreay fuzzy.

Reply:
> i suggest we can remove the return value since the meaning is alreay fuzzy.

done